crates/re_time_panel/src/lib.rs

```diff
@@ -574,7 +574,11 @@ impl TimePanel {
             item_response: response,
             body_response,
         } = ListItem::new(ctx.re_ui, text)
-            .with_icon(&re_ui::icons::ENTITY)
+            .with_icon(if tree.entity.components.is_empty() {
+                &re_ui::icons::ENTITY_EMPTY
+            } else {
+                &re_ui::icons::ENTITY
+            })
```

Comment:
There is an inconsistency here: the data UI uses `DataStore::all_components` which is timeful, while this uses the `EntityTree` which is timeless.

I.e. scrubbing the time cursor might change the icon that appears in the blueprint panel and elsewhere, while that won't happen in the time panel.

I think that's fine (unless we want both to be timeless), but deserves a comment to specify that this was intentional at least.

Reply:
As discussed, both were actually timeless, but `instance_path_icon()` accepting a `Query` argument was very confusing. I changed it to accept a `Timeline` instead, which is all it needs.

Also, I made both the streams and the blueprint have their icons based on the current timeline (in my original commit, the streams icon was based on _all timeline_).